tests/integration/loadProcesses.test.ts

```diff
@@ -0,0 +1,18 @@
+import { expect } from 'chai'
+import processes from '../fixtures/mock-processes.js'
+import { loadProcesses } from '../../src/lib/process/index.js'
+
+describe('loadProcesses function', () => {
+  let res: any
```

Comment:
can we avoid `any` here if we know the `res` is a `Process.Response`

Reply:
sorry but this is something I normally do, ideally I use just JS, easier, faster for testing.